### PR TITLE
MF-129: Allergy Level Three Widget

### PR DESCRIPTION
**Pull request by denniskigen**
Screenshot 2020-01-27 at 11 51 26

figma link https://www.figma.com/file/YyVhWTNp89bNiOGadYrHBk/Patient-Chart-Master-current-file?node-id=6427%3A189644

### CODE REVIEW

File: src/widgets/allergies/allergy-card-level-three.component.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        .then(allergy => setAllergy(allergy.data))
+        .catch(createErrorHandler());
+
+      return () => abortController.abort();
+    }
+  }, [isLoadingPatient, patient]);
+
+  function displayAllergy() {
+    return (
+      <SummaryCard
```
Comment (fatmali) [inline src/widgets/allergies/allergy-card-level-three.component.tsx]: What happens if there are no allergies to display?

File: src/widgets/allergies/allergy-card-level-two.css [hunk 2]
```diff
@@ -98,6 +98,9 @@
 
 .allergyFooter {
   text-align: center;
+  color: var(--omrs-color-ink-medium-contrast);
+  font-family: "Work sans";
+  font-size: 17px;
```
Comment (fatmali) [inline src/widgets/allergies/allergy-card-level-two.css]: use rem instead of px

Comment (denniskigen) [reply src/widgets/allergies/allergy-card-level-two.css]: Noted

File: src/widgets/allergies/allergy-card-level-three.component.tsx (hunk 1 above)
Comment (denniskigen) [reply src/widgets/allergies/allergy-card-level-three.component.tsx]: Could this situation arise? This widget would only display after you've clicked on an existing allergy.

Comment (fatmali) [reply src/widgets/allergies/allergy-card-level-three.component.tsx]: ooh right, it's alright then.

**Pull request merged**